### PR TITLE
fix(refs T31929): Optimize addon loading

### PR DESCRIPTION
**Pull request by salisdemos**
**Ticket:** https://yaits.demos-deutschland.de/T31929

Loading the '"default" Tabs/Components on the import page and then add some addons later on triggered a rerendering of the already loaded components, which broke the dpDatepicker and had potential to brake other things aswell.
To prevent that now the Tabs only get rendered after the addon-request is fulfilled. In premature obedience, to make it scalable, I already added a promisall resolving...  

### How to review/test

As FPA goto
/verfahren/<procedure-id>/import#StatementFormImport

check if the Datepicker for Einreichungsdatum/Verfassungsdatum works

### PR Checklist

- [x] Link all relevant tickets
- [x] Move the tickets on the board accordingly

### CODE REVIEW

File: client/js/components/procedure/AdministrationImport/AdministrationImport.vue [hunk 1]
```diff
@@ -160,12 +165,17 @@ export default {
...
-    },
+        })
+    }
   },
 
   mounted () {
-    this.loadComponents('import.tabs')
+    const loadComponents = this.loadComponents('import.tabs')
+
+    Promise.allSettled([loadComponents])
```
Comment (spiess-demos) [inline client/js/components/procedure/AdministrationImport/AdministrationImport.vue:175]: Nitpick: In other places we place the functions inside the brackets, like so:

```suggestion
    Promise.allSettled([this.loadComponents('import.tabs')])
```
Personally i find this more concise, what do you think?

**Review by spiess-demos — DISMISSED** [review]
lgtm! (code review only)

File: client/js/components/procedure/AdministrationImport/AdministrationImport.vue [hunk 2]
```diff
@@ -160,12 +165,15 @@ export default {
...
           }
-      })
-    },
+        })
+    }
   },
 
   mounted () {
-    this.loadComponents('import.tabs')
+        Promise.allSettled([this.loadComponents('import.tabs')])
```
Comment (salisdemos) [inline client/js/components/procedure/AdministrationImport/AdministrationImport.vue:173]:
```suggestion
    Promise.allSettled([this.loadComponents('import.tabs')])
```

**Review by spiess-demos — APPROVED** [review]
still lgtm

**Pull request merged**